partner_company_type/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Company Type',
+    'summary': """
+        Adds a company type to partner that are companies""",
```

Comment:
This is adding blank whitespace. Please use single line strings, optionally concatenated if too long:

```
'summary': 'my long'
           'summary',
```